scripts/reports/trial-sessions-by-year.ts

```diff
@@ -0,0 +1,46 @@
+// usage: npx ts-node --transpile-only ./scripts/reports/trial-sessions-by-year.ts 2023 > ~/Desktop/2023-trial-sessions.csv
+
+import {
+  FORMATS,
+  formatDateString,
+} from '@shared/business/utilities/DateHandler';
+import { createApplicationContext } from '@web-api/applicationContext';
+
+const year = process.argv[2] || '2022';
```

Comment:
Just curious, why is the default 2022?

Reply:
We defaulted to 2022 because that's the year they originally wanted these stats for!

As for the time, it actually is midnight Eastern Time, which is the Court's timezone.

Reply:
👍